src/test/java/net/datafaker/base/PhoneNumberTest.java

```diff
@@ -121,11 +125,89 @@ void testPhone_CA() {
     @RepeatedTest(10)
     void testAllCellPhone_enPh() throws NumberParseException {
         final PhoneNumberUtil util = PhoneNumberUtil.getInstance();
-        String phoneNumber = new BaseFaker(new Locale("en_PH")).phoneNumber().phoneNumber();
+        String phoneNumber = new BaseFaker(new Locale("en", "PH")).phoneNumber().phoneNumber();
         Phonenumber.PhoneNumber proto = util.parse(phoneNumber, "PH");
         assertThat(util.isValidNumberForRegion(proto, "PH")).as(phoneNumber).isTrue();
     }
 
+    @ParameterizedTest
+    @MethodSource("generateLanguageAndRegionOfLocales")
+    void testAllPhoneNumberNational(String localeLanguage, Map<String, String> regions) throws NumberParseException {
+        final PhoneNumberUtil util = PhoneNumberUtil.getInstance();
+        for (Map.Entry<String, String> region : regions.entrySet()) {
+            int errorCount = 0;
+            final BaseFaker faker = new BaseFaker(new Locale(localeLanguage, region.getKey()));
+            for (int i = 0; i < 10; i++) {
+                String phoneNumber = faker.phoneNumber().phoneNumber();
+                Phonenumber.PhoneNumber proto = util.parse(phoneNumber, region.getValue());
+                if (!util.isValidNumberForRegion(proto, region.getValue())) {
+                    errorCount++;
+                }
+            }
+            assertThat(errorCount).isLessThan(250);
+        }
+    }
+
+    @ParameterizedTest
+    @MethodSource("generateLanguageAndRegionOfLocales")
+    void testAllPhoneNumberInternational(String localeLanguage, Map<String, String> regions) throws NumberParseException {
+        final PhoneNumberUtil util = PhoneNumberUtil.getInstance();
+        for (Map.Entry<String, String> region : regions.entrySet()) {
+            int errorCount = 0;
+            final BaseFaker faker = new BaseFaker(new Locale(localeLanguage, region.getKey()));
+            for (int i = 0; i < 10; i++) {
+                String phoneNumber = faker.phoneNumber().phoneNumberInternational();
+                Phonenumber.PhoneNumber proto = util.parse(phoneNumber, region.getValue());
+                if (!util.isValidNumberForRegion(proto, region.getValue())) {
+                    errorCount++;
+                }
+            }
+            assertThat(errorCount).isLessThan(250);
+        }
+    }
+    // Key("IND") in `map("IND", "IN")` is a region for Java's Locale class
+    // Value("IN") in `map("IND", "IN")` is a region for PhoneNumberUtil's `parse` method.
```

Comment:
```suggestion
    @ParameterizedTest
    @MethodSource("generateLanguageAndRegionOfLocales")
    void testAllPhoneNumberNational(Locale locale, String phoneNumberRegion) throws NumberParseException {
        final PhoneNumberUtil util = PhoneNumberUtil.getInstance();
        int errorCount = 0;
        final BaseFaker faker = new BaseFaker(locale);
        for (int i = 0; i < 10; i++) {
            String phoneNumber = faker.phoneNumber().phoneNumber();
            Phonenumber.PhoneNumber proto = util.parse(phoneNumber, phoneNumberRegion);
            if (!util.isValidNumberForRegion(proto, phoneNumberRegion)) {
                errorCount++;
            }
        }
        assertThat(errorCount).isLessThan(250);
    }

    @ParameterizedTest
    @MethodSource("generateLanguageAndRegionOfLocales")
    void testAllPhoneNumberInternational(Locale locale, String phoneNumberRegion) throws NumberParseException {
        final PhoneNumberUtil util = PhoneNumberUtil.getInstance();
        int errorCount = 0;
        final BaseFaker faker = new BaseFaker(locale);
        for (int i = 0; i < 10; i++) {
            String phoneNumber = faker.phoneNumber().phoneNumberInternational();
            Phonenumber.PhoneNumber proto = util.parse(phoneNumber, phoneNumberRegion);
            if (!util.isValidNumberForRegion(proto, phoneNumberRegion)) {
                errorCount++;
            }
        }
        assertThat(errorCount).isLessThan(250);
    }

```

Reply:
Oh, I forgot to remove that foreach loop